### DISCUSSION TITLE
No ‚releases‘ under Explore Activity

### DISCUSSION THREAD

**Discussion by rml9ty** [community/community#80802] [Mobile]
Despite having selected 'releases' only - and definitely having projects stared - there is nothing ever listed as releases _anymore_. It has been that way for a few weeks. In the past this worked perfectly fine. 

![image](https://github.com/community/community/assets/51538059/5e8c1c5f-e3a6-4a5a-8dc4-7e5c242f6a1d)
![image](https://github.com/community/community/assets/51538059/ba0ef167-4976-48d2-bbe9-121df02b9888)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.142.1 (118013759)
Device: iPhone 13 Pro (iOS 17.0)
</details>

**Comment by alcere** [comment]
Hi @rml9ty, thanks for reporting this issue. After some investigation, we think we found the problem and it might related to this [change](https://github.blog/changelog/2023-09-06-updates-to-your-github-feed/). A temporary solution would be to set your Feeds configuration from the web version at https://github.com/dashboard and not use the mobile app.

We have opened an internal issue and we will try to get it fixed as soon as possible. We apologize for the inconvenience.

**Comment by rml9ty** [comment]
Funnily enough, yesterday it was working for a few hours on mobile. Now it's as I initially described it again. Looking forward to a permanent fix. Cheers!

**Reply by rml9ty** [reply]
The problem is back somehow. Changed nothing settings-wise.

**Reply by alcere** [reply]
Hello @rml9ty, we apologize for the inconvenience. We are still working on this and it is on our roadmap, so we appreciate your patience.

**Reply by rml9ty** [reply]
The "fix" that seemed to have worked last time as well seems to be setting the default filter settings in the web interface. After that one can selectively turn them off by preference on mobile again.
